### ISSUE TITLE
breadcrumbs - Kalimdor

### ISSUE THREAD

**Issue by Chulain** [ccshiro/cc-buglist#731]
## Kalimdor

---
## Ashenvale

http://www.wowhead.com/quest=4581/kayneth-stillwind
not sure

---

http://www.wowhead.com/quest=9533/a-helping-hand
2010/12/02 (Patch 4.0.3)
1    A Helping Hand(May be optional)
2    A Shameful Waste
3    Recover the Remains

prolly breadcrumb (found comment on part 2)

---

http://www.wowhead.com/quest=1056/journey-to-stonetalon-peak
2010/07/08 (Patch 3.3.5)
I've confirmed that this quest is not available if you've accepted or completed Reclaiming the Charred Vale. I'm trying to wrap up all quests on a high level character and had the quest Reclaiming the Charred Vale in my quest list and this one was not available. When I abandoned it, this quest immediately became available. If you've already done the charred vale quest, you're out of luck. Hope this helps.

---

http://www.wowhead.com/quest=9428/report-to-splintertree-post
2008/09/25 (Patch 2.4.3)
So far I have found the following to be true...
This quest is Unavailable to the Undead race.
This quest is Available to the Blood Elf race.

 2008/10/30 (Patch 3.0.2)
Not available to Troll race either, which is a bit ironic considering the rep for Darkspear Trolls. My suspicion is that it's only for Blood Elfs, but no hard proof on that.

2009/11/20 (Patch 3.2.2)
This quest can only be complited by Blood Elves. Taurens cant get this quest.

2009/02/26 (Patch 3.0.9)
This quest does count towards the Loremaster of Kalimdor achievement. I was able to complete it at level 80 (Blood Elf Rogue), even though I'd already done all the Splintertree Post quests.

there was one post saying it's available for taurens but it had a score of 0

---

http://www.wowhead.com/quest=1021/vile-satyr-dryads-in-danger
prolly not a breadcrumb tho

---

## Azshara

http://www.wowhead.com/quest=3504/betrayed
not sure

http://www.wowhead.com/quest=3421/return-trip
not a breadcrumb but thought u might wanna check it out

---
## Azuremyst Isle

http://www.wowhead.com/quest=9623/coming-of-age
(probably)

http://www.wowhead.com/quest=9313/travel-to-azure-watch
not sure

---

http://www.wowhead.com/quest=9505/the-prophecy-of-velen
2008/03/06 (Patch 2.3.3)
This is a Draenei only quest. However you can skip this quest and go right to Admiral Odesyus and pick up the quest chain at A Small Start if you're non Draenei. Not sure it works that way if you're Draenei.

---

http://www.wowhead.com/quest=10428/the-missing-fisherman
2012/09/16 (Patch 5.0.5)
Breadcrumb quest if you have already done All That Remains.

---

http://www.wowhead.com/quest=9625/elekks-are-serious-business
 2011/09/16 (Patch 4.2.2)
So, now the question is: What other breadcrumb quests are there that offer more exp/gold than the Hero's Call quests?

---

http://www.wowhead.com/quest=9314/word-from-azure-watch
not sure

---

## Bloodmyst Isle

http://www.wowhead.com/quest=9632/newfound-allies
could be a breadcrumb for the next quest, which is a breadcrumb

http://www.wowhead.com/quest=10064/talk-to-the-hand
probably a breadcrumb

---

http://www.wowhead.com/quest=9668/report-to-exarch-admetius
(probably)

---

http://www.wowhead.com/quest=10063/explorers-league-is-that-something-for-gnomes
https://youtu.be/_UzDZEAhfbw
(timestamp didnt work) 2:22 he finishes this quest but you can take the next part before doing so

---

http://www.wowhead.com/quest=965/the-tower-of-althalaxx
(not sure)

---

## Darnassus

http://www.wowhead.com/quest=730/trouble-in-darkshore
2007/07/31 (Patch 2.1.3)
This is an optional lead in quest. You can talk to Hollee without this one, but then you miss out on a little experience and IF rep.

---

http://www.wowhead.com/quest=2519/the-temple-of-the-moon
2009/07/18 (Patch 3.1.3)
If you're going for Loremaster, then you want to pick up this quest BEFORE http://www.wowhead.com/?quest=2518 because once you accept Tears of the Moon, you can't pick up this quest, losing out on a quest that counts toward the achievement.

---

http://www.wowhead.com/quest=6344/reminders-of-home
(most likely)
(quest is named "Nessa Shadowsong" in TBC)

---

## Desolace

http://www.wowhead.com/quest=6141/brother-anton
 2008/07/12 (Patch 2.4.2)
I can confirm this. I got Down the Scarlet Path without doing this quest.

---

http://www.wowhead.com/quest=1361/regthar-deathgate
 2008/05/13 (Patch 2.4.2)
Do I need to complete Regthar Deathgate in order to get this quest? Or is that one of those trivial quests intended to send you across the world, with the slight experience bonus?
 2008/06/30 (Patch 2.4.2)
you can get the quest without doing the "preq" for it

(found comments on next part)

---

http://www.wowhead.com/quest=823/report-to-orgnil
(most likely)

---

## Dustwallow Marsh

http://www.wowhead.com/quest=11225/the-hermit-of-witch-hill
2010/08/26 (Patch 3.3.5)
It appears this quest is _not_ available if you have already proceeded to do the follow-ups with Jarl. :(

---

http://www.wowhead.com/quest=11177/deprecated-the-hermit-of-swamplight-manor
2010/03/13 (Patch 3.3.2)
It seems that if you have already done Marsh Frog Legs, this quest is not available to you.

---

http://www.wowhead.com/quest=11212/tabethas-farm
http://www.wowhead.com/quest=11213/check-up-on-tabetha
(alliance & horde)
2007/12/26 (Patch 2.3.0)
This may seem like a worthless quest, but the truth is if you do this quest before you see her house (meaning if you go there and get the quest before you get this one) then you will find as I said more quest to do for her. So take the free xp and get some more quest to boot! Coords confirmed above 45,56

---

http://www.wowhead.com/quest=1260/deprecated-morgan-stern
2010/01/18 (Patch 3.3.0)
You can not do this quest if you have already completed "Mudrock Soup and Bugs" from "Morgan Stern"

---

http://www.wowhead.com/quest=11136/a-disturbing-development
2010/03/13 (Patch 3.3.2)
It seems that if you have already done Defias in Dustwallow?, then this quest is no longer available to you.
(quest that comes next)

---

## Felwood

http://www.wowhead.com/quest=5249/deprecated-to-winterspring
http://www.wowhead.com/quest=5250/starfall
2010/03/28 (Patch 3.3.3)
Also worth to note that neither "Starfall" and to "Winterspring" quest wont be avaiable if you start "The ruins of Khel'teril" as the next quest are avaiable without completing those.

---

http://www.wowhead.com/quest=8465/deprecated-speak-to-salfa
(not sure)

---

http://www.wowhead.com/quest=8462/deprecated-speak-to-nafien
breadcrumb but doesnt lead to a specific quest, just in case

---

http://www.wowhead.com/quest=854/journey-to-the-crossroads
breadcrumb but doesnt lead to a specific quest, just in case

---

## Orgrimmar

http://www.wowhead.com/quest=1145/the-swarm-grows
2007/03/22 (Patch 2.0.10)
To clarify, this is a lvl30+ quest that actually starts in the Crossroads. You have to talk to the troll in XR to get the followup to go to Thousand Needles, although you don't have to have done his other quests to get this. It's a rather strange quest chain, considering you don't really ever come by XR once you're lvl30. Worth the detour though.
next part also breadcrumb: http://www.wowhead.com/quest=1146/the-swarm-grows

---

http://www.wowhead.com/quest=2981/a-threat-in-feralas
2010/10/23 (Patch 4.0.1)
If you have completed the follow up quest The Ogres of Feralas, you will be unable to pick up this quest. Sad, because this DOES count towards Loremaster of Kalimdor.

---

http://www.wowhead.com/quest=235/the-ashenvale-hunt
http://www.wowhead.com/quest=742/the-ashenvale-hunt
http://www.wowhead.com/quest=6382/the-ashenvale-hunt

1 for each city, breadcrumb but doesnt lead to a specific quest, just in case

---

http://www.wowhead.com/quest=8276/taking-back-silithus
2010/07/31 (Patch 3.3.5)
This is one of the "talk to" quests, not necessary to do follow up. If you complete next quest before taking this one, you will lose +1 quest done for achievement.

---

http://www.wowhead.com/quest=4494/march-of-the-silithid
(probably)

---

## Silithus

http://www.wowhead.com/quest=9415/report-to-marshal-bluewall
http://www.wowhead.com/quest=9416/report-to-general-kirika
(alliance & horde)
2010/03/05 (Patch 3.3.2)
Seems that this quest is not required in order to do get the follow-up (Scouring the Desert), and becomes unavailable if you've completed that one.

---

## Stonetalon Mountains

http://www.wowhead.com/quest=5881/calling-in-the-reserves
probably a breadcrumb but dont know if it leads to another quest

http://www.wowhead.com/quest=1061/the-spirits-of-stonetalon
2010/09/02 (Patch 3.3.5)
This is a breadcrumb quest and cannot be picked up once you have completed/picked up Goblin Invaders.

---

http://www.wowhead.com/quest=6542/report-to-kadrak
http://www.wowhead.com/quest=6541/report-to-kadrak
2010/10/10 (Patch 3.3.5)
Not a pre-req but once "The Warsong Reports" is complete "Report to Kadrak" is unavailable.

---

## Tanaris

http://www.wowhead.com/quest=8922/a-supernatural-device
(not sure)

http://www.wowhead.com/quest=2872/stoleys-debt
2008/04/21 (Patch 2.4.1)
This quest becomes available when you complete Stoley's other quests. It is not required to get Stoley's Shipment (the next quest in the chain).

---

## Teldrassil

http://www.wowhead.com/quest=918/timberling-seeds
(not sure)

http://www.wowhead.com/quest=2159/dolanaar-delivery
breadcrumb i guess but dont know if it leads to any other Q
## The Barrens

http://www.wowhead.com/quest=1483/deprecated-ziz-fizziks
2009/05/18 (Patch 3.1.1)
It seems to me that this quest is an optional lead-in to Super Reaper 6000, and thus is not available if you have already done that quest.

---

http://www.wowhead.com/quest=886/the-barrens-oases
2007/06/22 (Patch 2.1.2)
It is possiable to skip this first part of the quest and pick the second part "The Forgotten Pools" up from Tonga Runetotem in Crossroads. I just did it!

---

http://www.wowhead.com/quest=6382/the-ashenvale-hunt
http://www.wowhead.com/quest=742/the-ashenvale-hunt
http://www.wowhead.com/quest=235/the-ashenvale-hunt

2008/11/04 (Patch 3.0.2)
Jorn Skyseer's version of this quest is not available until you complete Ishamuhale, which is in the middle of a fairly lengthy quest chain itself. You can get this quest from Org or TB as well, but Jorn's version gives about 500 more experience upon turn-in. Also, you don't need to take any of these three prerequisites to start the second part in Ashenvale.

---

http://www.wowhead.com/quest=860/sergra-darkthorn
2010/10/28 (Patch 4.0.1)
This appears to just be a cookie-crumb quest. I've completed the other steps in the quest and cannot pick this one up. Hope that helps someone out there!
it does indeed

---

## Thousand Needles

http://www.wowhead.com/quest=1179/the-brassbolts-brothers
not sure if it leads to another quest tho

http://www.wowhead.com/quest=4542/message-to-freewind-post
2010/06/04 (Patch 3.3.3)
This quest cannot be done if you've done the two quests after it in the chain.

---

http://www.wowhead.com/quest=5361/family-tree
breadcrumb, dont know if it leads to any other quest tho

http://www.wowhead.com/quest=2769/the-brassbolts-brothers
i remember doing the next part on retail (gahz'rilla) and i highly doubt i picked up this pre quest
most likely a breacrumb

---

## Thunder Bluff

http://www.wowhead.com/quest=1130/melor-sends-word
most likely

---

http://www.wowhead.com/quest=1064/deprecated-forsaken-aid
http://www.wowhead.com/quest=1065/deprecated-journey-to-tarren-mill

same questline, dont think they're breadcrumbs altho im not sure
third part wants you to go to tarren mill

---

http://www.wowhead.com/quest=4243/chasing-a-me-01
 2010/11/19 (Patch 4.0.3)
Is anyone aware of other pre-requisites to this quest or anything that would lock this quest out?
According to my wowhead profile, I haven't done this quest yet; however, it's not on offer from Karna.

don't think it's a breadcrumb but why wouldn't he be able to get it then? i'll leave it to you

---

## Winterspring

http://www.wowhead.com/quest=6028/the-everlook-report
not sure if it leads to another quest tho

---

http://www.wowhead.com/quest=4907/tinkee-steamboil
2008/05/14 (Patch 2.4.2)
You won't be able to pick up this one until you've completed the quest Return to Tinkee, which itself is part of a series beginning with Broodling Essence. Whether this is an optional or a required prerequisite for Egg Freezing, I'm not certain, but the next step after freezing the eggs in the Rookery is Egg Collection.

2009/01/21 (Patch 3.0.8)
As of 3.0.8, this lead-in quest no longer seems to count for Loremaster of Kalimdor.

---

http://www.wowhead.com/quest=6030/duke-nicholas-zverenhoff
not sure if it leads to another quest tho

---

http://www.wowhead.com/quest=6605/a-strange-one
2006/11/16 (Patch 1.12.2)
I don't think that this quest is step one in this series. I just received the quest Well of Corruption in Felwood and I have never been to Winterspring with this character.
2009/01/24 (Patch 3.0.8)
That's because it's one of those optional "go over to that other zone and talk to NPC X" ....

---

http://www.wowhead.com/quest=6603/trouble-in-winterspring
2009/12/29 (Patch 3.3.0)
It seems that this quest becomes unavailable if you have done the follow-up quest (Threat of the Winterfall) first.

---

http://www.wowhead.com/quest=6604/enraged-wildkin
2010/01/24 (Patch 3.3.0)
This quest is not available if you have done the followup first.

**Comment by Chulain** [comment]
https://github.com/ccshiro/cc-buglist/issues/730
